tests/scene/test_viewport.h

```diff
@@ -1935,6 +1935,142 @@ TEST_CASE("[SceneTree][Viewport] Embedded Windows") {
 	memdelete(w);
 }
 
+TEST_CASE("[SceneTree][Viewport] Camera3D override") {
+	SubViewport viewport;
```

Comment:
```suggestion
	SubViewport *viewport = memnew(SubViewport);
```
Use pointers for object types